app/assets/stylesheets/tamu.scss

```diff
@@ -0,0 +1,74 @@
+#header-navbar .navbar-brand {
+    background: transparent "tamu-logo-with-bar.png" no-repeat top left;
+    // background: transparent url(/assets/tamu/tamu-logo-with-bar.png) no-repeat top left;
+	margin-top: 10px;
+    margin-bottom: -10px;
+    
+}
+
```

Comment:
The tabbing is inconsistent.

I looked at the other SCSS files and I am seeing 2-space tabbing.
Can you update this entire file to consistently use 2-space tabbing?

Reply:
done